tests/System.Text.CaseFolding.Tests/Unicode/CharTests.cs

```diff
@@ -0,0 +1,1278 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text.CaseFolding;
+using System.Text;
+using Xunit;
+
+namespace System.Text.CaseFolding.Tests
+{
+    // The tests come from CoreFX tests: src\System.Runtime\tests\System\
+    public class CharTests
```

Comment:
A lot of these tests don't seem to be testing the new APIs. Why do we have them here?

Reply:
The file was copied from CoreFX. I have used some of them, but not sure that others should not be adapted to better cover the code.

Reply:
I think this whole class is unnecessary and should be removed. The only place you use it is to find the assembly name.
```C#
Stream stream = typeof(CharTests).GetTypeInfo().Assembly.GetManifestResourceStream(fileName);
```

I believe this can be written using any other existing test class:
```C#
Stream stream = typeof(CharUnicodeInfoTestData).GetTypeInfo().Assembly.GetManifestResourceStream(fileName);
```

If it isn't testing any of the new APIs being added in this PR, it doesn't make sense to copy this class here.